modular_zubbers/code/modules/security_deathrattle/security_deathrattle.dm

```diff
@@ -0,0 +1,18 @@
+//This file makes it so that security will always get a deathrattle.
+
+/datum/station_trait/deathrattle_department/security
+	weight = 0
+
+/datum/controller/subsystem/processing/station/SetupTraits()
+
+	. = ..()
+
+	var/datum/station_trait/deathrattle_department/security/existing_security_trait = locate() in station_traits
+	if(existing_security_trait)
+		return .
```

Comment:
Is this really needed? With the weight set to 0 this should never prop, no?

Reply:
true, but it's just in case:tm: